src/shortcuts.py

```diff
@@ -168,7 +168,7 @@ def add_shortcuts(widget, settings):
         Gtk.NamedAction.new("cavalier.prev-profile")))
 
 def change_mode(action, parameter, settings, diff):
-    modes = ['wave', 'levels', 'particles', 'bars']
+    modes = ['wave', 'levels', 'particles', 'bars', 'line']
```

Comment:
I decided to try again and finally I found a way to get the list of modes directly from GSettings. You can replace this line with
`modes = settings.get_range('mode')[1]`
so we'll not need to add new values to the list here manually.